packages/whats-new/src/style.scss

```diff
@@ -1,272 +0,0 @@
-@import "@automattic/typography/styles/fonts";
-@import "@automattic/onboarding/styles/mixins";
-@import "@automattic/calypso-color-schemes";
```

Comment:
Another color schemes import down (cc @jasmussen for sharing the celebration)

Reply:
Hooray!